main.go

```diff
@@ -2,6 +2,9 @@ package main
 
 import (
 	"fmt"
+	"os"
```

Comment:
looking at these import refactor I am wondering if you are using the right formatter. Are you using Visual Studio code ?
I had the same issue until I changed the default format tool.
in VSCode go to preferences and type `go format tool` then pick *gofmt* instead of the default `goretruns`

Reply:
hehe, yeah, VSCode, but using `goimports`. I don't have a strong opinion either way (I probably should though). But I don't mind reverting any import diffs in this PR.

Reply:
I don't mind either way, it is just sometimes make it hard to focus on the real changes.